content/en/docs/refguide/modeling/application-logic/workflows/boundary-events.md

```diff
@@ -0,0 +1,105 @@
+
+- Interrupting Boundary Events: When these events are triggered, they interrupt the normal path of the activity they are attached to. The activity stops, and the process flow is redirected to the boundary event's outgoing sequence path. As per BPMN2.0 specification, interrupting boundary events are visualised as two solid circles.
+
+## Types
+
+Within the current release of the boundary events, Studio Pro has support for one type of boundary event being the timer boundary event.
+
+|Image | Type | Description|
+|-------- | -------- | --------|
+| {{< figure src="/attachments/refguide/modeling/application-logic/workflows/workflow-elements/timer.png" width="50px" alt="Timer Event" class="no-border" >}} | [Timer Events](/refguide/timer-event) | The Timer event can also be used as a boundary event.
+
+## Supporting activities
+
+The following activities have support for the new boundary events.
+
+| Graphic                                                     | Element                           |
+| ----------------------------------------------------------- | --------------------------------- |
+| {{< figure src="/attachments/refguide/modeling/application-logic/workflows/workflow-elements/wait-for-notification.png" alt="Wait for notification activity" class="no-border" >}} | [Wait for notification](/refguide/wait-for-notification/)
+| {{< figure src="/attachments/refguide/modeling/application-logic/workflows/workflow-elements/user-task.png" width="60px" alt="User Task" class="no-border" >}} | [User task](/refguide/user-task/) 
+| {{< figure src="/attachments/refguide/modeling/application-logic/workflows/workflow-elements/multi-user-task.png" width="60px" alt="Multi-user task" class="no-border" >}} | [Multi-user task](/refguide/multi-user-task/)
+| {{< figure src="/attachments/refguide/modeling/application-logic/workflows/workflow-elements/call-microflow.png" alt="Call Microflow" class="no-border" >}} | [Call microflow](/refguide/call-microflow/)
+| {{< figure src="/attachments/refguide/modeling/application-logic/workflows/workflow-elements/call-workflow.png" alt="Call Workflow" class="no-border" >}} | [Call workflow](/refguide/call-workflow/)
+
+## Getting started
+
+To get started with the new boundary events feature, these can be enabled by opening Studio Pro's settings and navigating to new features (this setting is only applicable during the beta).
+
+- Windows: Preferences -> New Features -> Enable non-interrupting timer boundary events (beta)
+- Mac (Beta): Preferences -> New Features -> Enable non-interrupting timer boundary events (beta)
+
+{{< figure src="/attachments/refguide/modeling/application-logic/workflows/workflow-how-to-boundary-events/boundary-events-experimental-option.png" alt="Experimental option" width="450" class="no-border">}}
+
+## Adding Timer Boundary Events
```

Comment:
Since we will have multiple boundary event types in the future does it make sense to make this section more generic for all boundary events?
The first 2 steps are quit generic. Only the last step is specific for timer events, We would leave that for now or also refer to the specific event configuration page (Timer in this case)
@Yiyun333 WDYT?

Reply:
@PaulJongenMx This section should be more generic I think. I will say "Adding a Boundary Event" in this case. The instructions should be generic as well and when needed, we can use Timer as an example. 

Please see my suggestion below: 

> ## Adding a Boundary Event
> 
> To add a boundary event to the above-listed activities, choose one of the following ways:
> 
> * Select an event from the **Events** section in the workflow **Toolbox** (currently only Timer is available) and drag it onto one of the activities listed above. Double-click the event to open its properties dialog box.
> 
> {{< figure src="/attachments/refguide/modeling/application-logic/workflows/workflow-how-to-boundary-events/drag-drop.png" alt="Adding boundary events through drag and drop" width="200" class="no-border">}}
> 
> * Double-click one of the above-listed activities to open its properties dialog box, go to the **Events** tab, and in the > **Boundary events** section, click **New** to add a new boundary event (currently only Timer is available).
> 
> {{< figure src="/attachments/refguide/modeling/application-logic/workflows/workflow-how-to-boundary-events/edit-dialog.png" alt="Adding boundary events through edit dialog" width="450" class="no-border">}}
> 
> For more information on how to configure the properties of a timer boundary event, see the [Properties](/refguide/timer/#properties) section in *Timer*.

I removed all the numbers in the instructions. I don't think they are needed because they can be confusing too. I can remove them from the screenshot as well.

Reply:
I used yiyun's suggestion